core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -124,6 +124,14 @@ angular.module('oppia').controller('RteHelperModalController', [
           $uibModalInstance.dismiss('cancel');
           return;
         }
+        var resampledFile = (
+          ImageUploadHelperService.convertImageDataToImageFile(svgFile));
+        const HUNDRED_KB_IN_BYTES = 100 * 1024;
+        if (resampledFile.size > HUNDRED_KB_IN_BYTES) {
+          AlertsService.addWarning('The SVG file generated exceeds 100 KB.');
+          $uibModalInstance.dismiss('cancel');
```

Comment:
Hm, if this case should occur, what is the user expected to do (i.e. how can they represent their math expression in the lesson)?

Reply:
In this case the user has to split the equation as 2 expressions.(Because anyway the equations are rendered inline)
Ex: If the long expression is 1+ 2+ 3 + 4+ 5 + 6 + 7. the user can split it as 1 + 2+ 3 + 4 and + 5 + 6 + 7 .

Reply:
I see. Do you think the user would be aware of this? If not, it may be worth explaining to them how to do it -- e.g. in the error message itself or as part of a different "info" message. What do you think?

Reply:
done, added that info